gbpservice/nfp/service_vendor_agents/vyos/agent/src/auth_server.conf

```diff
@@ -0,0 +1,18 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+http://10.30.120.97:5000/
+services
+neutron
```

Comment:
Where are these creds used? Looks like these are Openstack creds. They will not work on a different setup right?